android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISnackbar.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2012 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+package ti.modules.titanium.ui.widget;
+
+import android.view.View;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import org.appcelerator.kroll.KrollDict;
+import org.appcelerator.titanium.TiC;
+import org.appcelerator.titanium.proxy.TiViewProxy;
+import org.appcelerator.titanium.util.TiConvert;
+import org.appcelerator.titanium.view.TiUIView;
+
+public class TiUISnackbar extends TiUIView
+{
+	private static final String TAG = "TiUISnackbar";
+
+	public TiUISnackbar(TiViewProxy proxy)
+	{
+		super(proxy);
+	}
```

Comment:
Is this required if nothing is overwritten?

Reply:
yes, it complains about a missing default constructor otherwise